doc/llm.md

```diff
@@ -0,0 +1,27 @@
+---
+page:
+  image: llm.png
+---
+
+# Running local LLM using ollama and open-webui
+
+While `services-flake` is generally used for running services in a *development* project, typically under a source code checkout, you can also write flakes to derive an end-user app which runs a group of services, which then can be run using `nix run` (or installed using `nix profile install`):
```

Comment:
I'd really expand this paragraph to say something about AI chatbots and get the user excited about running them locally without needing to do network access (outside of downloading models).